### ISSUE TITLE
Used BCE - Loss but no sigmoid activation

### ISSUE THREAD

**Issue by Gunnvant** [yunlongdong/FCN-pytorch-easiest#7]
You have used BCE loss in your train file but I don't see you using sigmoid activation. Will that not affect anything as we use sigmoid activation when using BCE loss and Softmax when using Cross Entropy loss
